mod-source-record-storage-server/src/main/java/org/folio/dao/RecordDaoImpl.java

```diff
@@ -443,13 +449,16 @@ public RecordDaoImpl(final PostgresClientFactory postgresClientFactory) {
       .offset(offset)
       .limit(limit)
       .getSQL(ParamType.INLINED);
-    return getCachecPool(tenantId).rxBegin()
+
+    return getCachePool(tenantId)
```

Comment:
Same as above, no longer in a transaction. Should be:

```
    return getCachedPool(tenantId)
      .rxGetConnection()
      .flatMapPublisher(conn -> conn.rxBegin()
        .flatMapPublisher(tx -> conn.rxPrepare(sql)
          .flatMapPublisher(pq -> pq.createStream(1)
            .toFlowable()
            .map(this::toRow)
            .map(this::toSourceRecord))
          .doAfterTerminate(tx::commit)));
```

Reply:
@wwelling done